src/core_seaice/shared/mpas_seaice_forcing.F

```diff
@@ -1685,21 +1698,35 @@ end subroutine init_data_iceberg_forcing
 
   subroutine data_iceberg_forcing(&
        streamManager, &
-       domain, &
-       simulationClock)
+       domain)
 
     type (MPAS_streamManager_type), intent(inout) :: streamManager
 
     type (domain_type) :: domain
 
-    type (MPAS_clock_type) :: simulationClock ! dc is this being used?
+    ! for debugging
+    ! type (MPAS_time_type) :: currentForcingTime
+    ! character(len=strKIND) :: currentForcingTimeStr
+
+    logical, pointer :: &
+         config_do_restart
 
     real(kind=RKIND), pointer :: &
          config_dt
 
     call mpas_pool_get_config(domain % configs, 'config_dt', config_dt)
 
-    ! use the forcing layer to get data
+    ! for debugging
+    ! ! get the current time
+    ! call MPAS_forcing_get_forcing_time(&
+    !    seaiceForcingGroups, &
+    !    "seaice_data_iceberg_forcing_monthly", &
+    !    currentForcingTime)
+
+    ! call MPAS_get_time(currentForcingTime, dateTimeString=currentForcingTimeStr)
+    ! call MPAS_log_write('Get Data icebergs at: '//trim(currentForcingTimeStr))
+
```

Comment:
@akturner - These commented out lines were added by @jonbob during debugging to make sure the forcing and simulation times were aligned correctly. While they're extra commented out lines in the code, they were useful to have, and could be useful in other situations. Another option would be to have such debugging messages implemented more robustly using the framework (for another day).

Reply:
@darincomeau: Thats fine, but maybe a more detailed comment about their purpose?